python/triton/language/semantic.py

```diff
@@ -1382,12 +1390,15 @@ def assert_dtypes_valid(lhs_dtype, rhs_dtype, target):
     # hip for now converts fp8 to fp16 for mixed input
     if is_hip():
         fp8_supported = gpu_matrix_core_version() == 3
+        # gfx940 data type
+        lhs_hip_fp8 = lhs.type.scalar.is_fp8e4b8() or lhs.type.scalar.is_fp8e5b16()
+        rhs_hip_fp8 = rhs.type.scalar.is_fp8e4b8() or rhs.type.scalar.is_fp8e5b16()
         lhs_fp8 = lhs.type.scalar.is_fp8()
```

Comment:
if `lhs_fp8` and `rhs_fp8` are not needed, maybe we can just remove them?

Reply:
They are used in line 1399 and line 1401, respectively